src/main/java/games/strategy/triplea/ui/menubar/LobbyMenu.java

```diff
@@ -423,4 +434,10 @@ private void addExitMenu(final JMenu parentMenu) {
       parentMenu.add(menuFileExit);
     }
   }
+
+  private static Date toDate(final TemporalAmount amount) {
```

Comment:
Reading through this a second time, I realized that this method is doing more than simply converting a `TemporalAmount` to a `Date`.  `TemporalAmount` represents a duration, while a `Date` is an absolute point in time, so it's really adding the given duration to "now," and then doing the conversion.  That is, this function isn't "pure" in the sense that it has a hidden dependency on global state.  Capturing that in the method name would be helpful.

Is there a better method name here?  `nowPlusDuration()`?  `nowOffsetBy()`?  Those could be a bit too generic in this context.  Maybe something more context-dependent like `getBanExpirationDate()`?  Meh, I hate naming things...

Reply:
Did think about `getBanExpirationDate`, but as this method can not just be exclusively use by ban actions I droppen "Ban" from the name...